spec/rails_helper.rb

```diff
@@ -10,6 +10,7 @@
 require File.join(File.dirname(__FILE__), 'support/factory_girl')
 require File.join(File.dirname(__FILE__), 'support/database_cleaner')
 require File.join(File.dirname(__FILE__), 'helpers/fill_in')
+require File.join(File.dirname(__FILE__), 'helpers/sign_in')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.